setup/tue-functions.bash

```diff
@@ -47,16 +47,21 @@ function _tue-git-branch-clean
     local force_remove
     local error_code
     local stale_branches
+    local repo_path
+    repo_path="$PWD"
     local repo
-    repo="$PWD"
+    repo=$(basename "$repo_path")
 
     if [ -n "$1" ]
     then
         if [ "$1" == "--force-remove" ]
+        then
+            force_remove=true
+        elif [ -z "$2" ] && [ "$1" == "true" ]
```

Comment:
What's with this? Why do you want to have a parameter "`true`"? Having just an option like `--force-remove` is more than enough

Reply:
Because you pass on true from `tue-git-branch-clean` to `_tue-git-branch-clean`. So either this way, or just pass on all arguments to the child function.

Reply:
I think passing on all the arguments to the child function is cleaner. This looks ugly. I definitely made a mistake by parsing the argument in the parent function.

Reply:
However a simple check in the parent function is required else there will be tons of printed statements on the screen.